### PR TITLE
Add note about ACL for anonymous connections

### PR DESCRIPTION
**Pull request by KNXBroker**
Added ACL requirement at anonymous description (in addition to 'Known issues and limitations').

New users (e.g. myself) do not read the complete documenation bevor they start using the add-on. If you do not read the docu till the bottom, you don't know that ACL is required for anonymous connections.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @KNXBroker, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#1645).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: mosquitto/DOCS.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ logins:
 
 ### Option: `anonymous`
 
-Allow anonymous connections. If logins are set, the anonymous user can only read data.
+Allow anonymous connections. If logins are set, the anonymous user can only read data. Since version 4.1 of the add-on, an explicit ACL definition is required for anonymous connections [see these instructions](#access-control-lists-acls).
```
Comment (ludeeus) [inline mosquitto/DOCS.md:57]:
```suggestion
Allow anonymous connections. If logins are set, the anonymous user can only read data. An explicit ACL definition is required for anonymous connections [see Access Control Lists (ACLs)](#access-control-lists-acls).
```

Documentation should reflect the current version, no need to specify.

**Comment by pvizeli** [comment]
Why do you not update the add-on to handle that automatically for the user?

**Comment by frenck** [comment]
I agree with @pvizeli, Although this does reflect the current state of affairs. So, let's add this now to make it match. And improve on it later?

**Comment by KNXBroker** [comment]
> So, let's add this now to make it match. And improve on it later?

I agree 😄

**Review by ludeeus — APPROVED** [review]

**Pull request merged**